### DISCUSSION TITLE
Add Button to Turn Code Coverage on/off

### DISCUSSION THREAD

**Discussion by abivelj** [forcedotcom/salesforcedx-vscode#5742] [Ideas]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
A button on the status bar that you can click that will turn on/off the code coverage so you don't have to go to the settings and make the change when you just want to run the test without code coverage.

**Describe alternatives you've considered**
Going to the settings and turning it on and off.

**Comment by abivelj** [comment]
I can't get the solution to run. I start the "launch extension" debug configuration, but when the extensions start to load I just get this error: "activating extension 'salesforcedx-vscode-apex' failed: undefined.

I'm usually developing in visual studio so I'm not as familiar with vs code development or vacode extensions, can anyone help?

**Comment by ntotten** [comment]
@abivelj Can you open a new issue for the activation problem and provide full output of the output window.

**Comment by jairzh** [comment]
I use this plugin to toggle **Code Coverage**

https://marketplace.visualstudio.com/items?itemName=rebornix.toggle
```
{
  "key": "cmd+k cmd+t",
  "command": "toggle",
  "when": "editorTextFocus",
  "args": {
    "id": "sfdc-test-converage",
    "value": [
      {
        "salesforcedx-vscode-core.retrieve-test-code-coverage": true
      },
      {
        "salesforcedx-vscode-core.retrieve-test-code-coverage": false
      }
    ]
  }
} 
```
But sometime, I can not remember its on or off, if there is a button on the status bar that will better.
